tests/test_ungriddeddata.py

```diff
@@ -252,3 +253,10 @@ def test_check_convert_var_units(data_scat_jungfraujoch):
             ratio = ratio[~np.isnan(ratio)]
 
             npt.assert_allclose(actual=[ratio.mean(), ratio.std()], desired=[fac, 0], atol=1e-20)
+
+def test_from_single_station_data():
+    stat = FAKE_STATION_DATA["station_data1"]
+    d = ungriddeddata.UngriddedData.from_station_data(stat)
+    data0 = stat.ec550aer
+    data1 = d.all_datapoints_var("ec550aer")
+    npt.assert_allclose(data0, data1, atol=1e-20)
```

Comment:
The test itself looks right, but black is going to complain about the formatting (no line at the end of the file).

Make sure that you format the code with black and isort.
Send me a message if you need more instructions.